app/models/views/confirmation/result.rb

```diff
@@ -28,7 +28,13 @@ def benefits_passed?
 
       def income_passed?
         if application_type_is?('income')
-          convert_to_pass_fail(%w[full part].include?(@application.outcome).to_s)
+          if @application.waiting_for_evidence?
+            I18n.t('activemodel.attributes.views/confirmation/result.income_evidence')
+          elsif @application.waiting_for_part_payment?
+            I18n.t('activemodel.attributes.views/confirmation/result.income_part')
+          else
+            convert_to_pass_fail(%w[full part].include?(@application.outcome).to_s)
+          end
```

Comment:
Just to avoid the multi if clause, another way to write this could be:

```ruby
path = 'activemodel.attributes.views/confirmation/result'
income_evidence = I18n.t('income_evidence', scope: path)
part_payment = I18n.t('income_part', scope: path)

return income_evidence if @application.waiting_for_evidence?
return part_payment if @application.waiting_for_part_payment?
convert_to_pass_fail(%w[full part].include?(@application.outcome).to_s)
```